packages/core/src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,169 @@
+      this._id = uniqueId(`select_${this.props.name}_`);
+    }
+    return this._id;
+  }
+
+  render() {
+    /* eslint-disable prefer-const */
+    let {
+      // Using let rather than const since we sometimes rewrite id
+      className,
+      errorMessage,
+      fieldClassName,
+      fieldRef,
+      focusTrigger,
+      hint,
+      inversed,
+      label,
+      labelClassName,
+      options,
+      requirementLabel,
+      size,
+      ...selectProps
+    } = this.props;
+    /* eslint-enable prefer-const */
+
+    const classes = classNames(className);
+    const fieldClasses = classNames(
+      'ds-c-field',
+      { 'ds-c-field--inverse': inversed },
+      size && `ds-c-field--${size}`,
+      fieldClassName
+    );
+
+    const optionElements = options.map(option => (
+      <option key={option.value}>{option.label}</option>
+    ));
+
+    return (
+      <div className={classes}>
+        <FormLabel
+          className={labelClassName}
+          component={'label'}
+          errorMessage={errorMessage}
+          fieldId={this.id()}
+          hint={hint}
+          requirementLabel={requirementLabel}
+          inversed={inversed}
+        >
+          {label}
+        </FormLabel>
+        <select
+          className={fieldClasses}
+          id={this.id()}
+          /* eslint-disable no-return-assign */
+          ref={focusTrigger ? loader => (this.loader = loader) : fieldRef}
+          /* eslint-enable no-return-assign */
+          {...selectProps}
+        >
+          {optionElements}
+        </select>
+      </div>
+    );
+  }
+}
+
+Dropdown.propTypes = {
+  /**
+   * Adds `aria-label` attribute if component renders a select
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * Additional classes to be added to the root element.
+   */
+  className: PropTypes.string,
+  /**
+   * Sets the initial selected state. Use this for an uncontrolled component;
+   * otherwise, use the `value` property.
+   */
+  defaultValue: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  /**
+   * Disables the entire field.
+   */
+  disabled: PropTypes.bool,
+  errorMessage: PropTypes.node,
+  /**
+   * Additional classes to be added to the select element
+   */
+  fieldClassName: PropTypes.string,
+  /**
+   * Access a reference to the `select` element
+   */
+  fieldRef: PropTypes.func,
+  /**
+   * Used to focus `select` on `componentDidMount()`
+   */
+  focusTrigger: PropTypes.bool,
+  /**
+   * Additional hint text to display
+   */
+  hint: PropTypes.node,
+  /**
+   * Applies the "inverse" UI theme
+   */
+  inversed: PropTypes.bool,
+  /**
+   * Label for the field
+   */
+  label: PropTypes.node.isRequired,
+  /**
+   * Additional classes to be added to the `FormLabel`.
+   */
+  labelClassName: PropTypes.string,
+  /**
+   * The field's `name` attribute
+   */
+  name: PropTypes.string.isRequired,
+  /**
+   * The list of options to be rendered.
+   */
+  options: PropTypes.arrayOf(
```

Comment:
I was considering renaming this prop from “choices” to “options” to differentiate the Dropdown prop from the Choicelist one. I was thinking it might be useful to do this because the props are slightly different, and “options” relates to the actually html element used in this component. That being said, it might be better to just keep the names the same too make it easier to convert between Choicelist and Dropdown. Do you have any thoughts? @line47 @phiden

Reply:
I don't have strong feelings either way - I think this is smart to think about. How much simpler does it make it by not switching the name from `choices` to `options`?

Reply:
The main impact I was thinking was if you were converting the old `ChoiceList` to the new `Dropdown`, you could avoid renaming that prop. However, you would still have to change the props like this: 
```
<ChoiceList choices={[{
   value: 1,
   label: 'first choice',
   checked: true,
}]}/>
```
becomes ->
```
<Dropdown choices={[{
   value: 1,
   label: 'first choice',
}]} value={1} />
```

The main reason why `Dropdown` has a new a `value/defaultValue` prop and has a slightly modified `choices/options` prop is because this prevents multiple options from being selected. 
I was considering keeping `choices` exactly the same and just throwing an error if multiple choices were selected, but I thought it would be better to just design the props so that only 1 selected option is allowed.

Reply:
The main consideration I was thinking of was if you were converting the old `ChoiceList` to the new `Dropdown`, you could avoid renaming that prop. However, you would still have to change the props like this: 
```
<ChoiceList choices={[{
   value: 1,
   label: 'first choice',
   checked: true,
}]}/>
```
becomes ->
```
<Dropdown choices={[{
   value: 1,
   label: 'first choice',
}]} value={1} />
```

The main reason why `Dropdown` has a new a `value/defaultValue` prop and has a slightly modified `choices/options` prop is because this prevents multiple options from being selected. 
I was considering keeping `choices` exactly the same and just throwing an error if multiple choices were selected, but I thought it would be better to just design the props so that only 1 selected option is allowed.

Reply:
The main consideration I was thinking of was if you were converting the old `ChoiceList` to the new `Dropdown`, you could avoid renaming that prop. However, you would still have to change the props like this: 
```
<ChoiceList choices={[{
   value: 1,
   label: 'first choice',
   checked: true,
}]}/>
```
becomes ->
```
<Dropdown choices={[{
   value: 1,
   label: 'first choice',
}]} value={1} />
```

The main reason why `Dropdown` has a new a `value/defaultValue` prop and has a slightly modified `choices/options` prop is because this prevents multiple options from being selected. 
I was considering keeping `choices` exactly the same and just throwing an error if multiple choices were selected, but I thought it would be better to just design the props so that only 1 selected option is allowed.

Reply:
That sounds great, thanks!

Reply:
I concur with @line47 about the prop name change. `options` is a clear prop to me, esp. since the rendered HTML is a series of `<option>` tags and we want to prevent multiple selection.